Wikipedia/Code/NotificationsCenterViewModel.swift

```diff
@@ -62,8 +77,10 @@ final class NotificationsCenterViewModel: NSObject {
     // MARK: - Public
     
     func refreshNotifications() {
+        isLoading = true
```

Comment:
I see what you mean about this loading state getting messy because importing and refreshing can step on each other a bit deeper within the remote notifications controllers. I'll give some thought to cleaning that area up.